content/docs/standard-library/io/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: IO
+weight: 5
+---
+The `IO`  section describes the standard component's to deal with input and output
+
+# Printer
+The `Printer` is a component used to print to stdout
```

Comment:
It prints given `data` and after printing is done it sends this exact message to it's `sig` outport